port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -70,22 +70,43 @@ async def _search_as_object(
     async def _calculate_entities(
```

Comment:
this function is getting big please move the inner function out with meaningful names